core/native-bridge.ts

```diff
@@ -530,21 +546,33 @@ const initBridge = (w: any): void => {
             const {
               data: requestData,
               type,
-              headers,
+              headers: requestHeaders,
             } = await convertBody(
               options?.body || body || undefined,
               optionHeaders['Content-Type'] || optionHeaders['content-type'],
             );
 
+            const nativeHeaders = {
+              ...requestHeaders,
+              ...optionHeaders,
+            };
+
+            if (platform === 'android') {
+              if (headers.has('User-Agent')) {
+                nativeHeaders['User-Agent'] = headers.get('User-Agent') as string;
```

Comment:
```suggestion
                nativeHeaders['User-Agent'] = headers.get('User-Agent');
```
I don't think the cast is needed